cpp-inference/inference_scaling_imagenet.cpp

```diff
@@ -92,16 +92,14 @@ void run_mnist(const std::string& model_name,
       for (int i=0; i<num_devices; i++ ) {
         std::string model_key = "resnet_model_" + std::to_string(i);
         std::string script_key = "resnet_script_" + std::to_string(i);
-        std::string device_key = device + ":" + std::to_string(i);
-        std::cout<<"Device Key " <<device_key <<std::endl<<std::flush;
-
+        std::string model_name = "./resnet50." + device + ".pt";
```

Comment:
Is this adding a case sensitivity to the code that isn't present elsewhere?

Reply:
I don't think so, but I can run `toupper` on `device` to make sure we don't need to adjust any other part of the code.